pkg/crinit/standalone/standalone.go

```diff
@@ -70,10 +70,16 @@ func NewCmdStandalone(cmdOut io.Writer, pathOptions *clientcmd.PathOptions, defa
 	opts := &standaloneClusterRegistryOptions{}
 
 	cmd := &cobra.Command{
-		Use:     "standalone CLUSTER_REGISTRY_NAME --host-cluster-context=HOST_CONTEXT",
+		Use:   "standalone <subcommand>",
```

Comment:
Add `<subcommand>` to the aggregated version too or remove it from here too. Either way is fine, but being consistent would be better.

Reply:
Good point. Oddly, the `<subcommand>` bit didn't appear when I was running the command, so I'll remove it and let the `cobra` machinery handle describing the behavior.